### PR TITLE
Update volunteers

### PR DESCRIPTION
**Pull request by nholden**
This PR adds headshots and titles for volunteers from the Google Drive (TJI Drive > About Us > tji team). The only existing photo I edited was for Shea, because the original photo was not a 1:1 aspect ratio, so I cropped it to fix the layout.

Closes https://github.com/texas-justice-initiative/website-nextjs/issues/82

### CODE REVIEW

File: data/volunteers.js [hunk 1]
```diff
@@ -9,11 +9,36 @@ export default [
     title: 'Front-End Developer',
     image: 'simi-damani.jpg',
   },
+  {
+    name: 'Nick Holden',
+    title: 'Software Engineer',
```
Comment (nholden) [inline data/volunteers.js]: I made this title up for myself. I wonder if it makes sense to consolidate some of these titles (for example, is differentiating between "software engineer," "senior software engineer," and "front-end developer" helpful?). Not a high priority for me though if it doesn't bother anyone else.

File: data/volunteers.js [hunk 2]
```diff
@@ -9,11 +9,36 @@ export default [
...
+    image: 'nick-holden.jpg',
+  },
+  {
+    name: 'Dashiel Lopez Mendez',
+    title: 'Software Engineer',
```
Comment (nholden) [inline data/volunteers.js]: There was no title in the image name in the Google Drive, so I Googled Dash and this seemed like an appropriate title. Would you prefer something else, @laconc?

File: data/volunteers.js [hunk 3]
```diff
@@ -9,11 +9,36 @@ export default [
...
+    image: 'daniel-olivares.jpg',
+  },
   {
     name: 'Jonathan Pascoe',
     title: 'Geographic Information Systems Professional',
     image: 'jonathan-pascoe.jpg',
   },
+  {
+    name: 'Athula Pudhiyidath',
+    title: 'Graduate Student Researcher',
```
Comment (nholden) [inline data/volunteers.js]: There was no title in the image name in the Google Drive, so I Googled Athula and this seemed like an appropriate title. Would you prefer something else, @athpud?

Comment (EvaRuthM) [reply data/volunteers.js]: Hi @nholden, sorry about the missing title -- please use "Data Scientist" for @athpud -- thanks!

File: data/volunteers.js (hunk 2 above)
Comment (EvaRuthM) [reply data/volunteers.js]: previously, we used "Infrastructure" for @laconc -- whichever he prefers works for me!

**Comment by nholden** [comment]
Thanks for the edits, @EvaRuthM! I updated them in https://github.com/texas-justice-initiative/website-nextjs/pull/94/commits/ea16c73c8d0f6cb8e7dde00ef0894860f7c0126a. And of course, we can change them at any time if we'd like.

**Pull request merged**